integr8ly/tutorial-web-app-operator
# Bump 0.0.12

Verification steps:
- Deploy/update a build of this operator (`docker.io/aidenkeatingrht/tutorial-web-app-operator:0.0.12`) to a cluster.
- Create a `webapp` CR if one doesn't already exist, ensure it doesn't have `WALKTHROUGH_LOCATIONS` set
- Ensure that the newly deployed Deployment for the tutorial web app is using:
  - The image: `quay.io/integreatly/tutorial-web-app:v2.9.0`
  - The default WALKTHROUGH_LOCATIONS: `https://github.com/integr8ly/tutorial-web-app-walkthroughs.git#v1.6.2`
